src/Confluent.Kafka/WatermarkOffsets.cs

```diff
@@ -33,7 +33,7 @@ public class WatermarkOffsets
         ///     The offset of the earlist message in the topic/partition.
         /// </param>
         /// <param name="high">
-        ///     The offset of the last committed message in the topic/partition.
+        ///     The offset of the last stored message in the topic/partition.
```

Comment:
No, dont! I was in the wrong, the watermark offsets have nothing to do with committed offsets. they are just the oldest and newest message offsets of a partition.